service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapper.java

```diff
@@ -254,16 +252,28 @@ private String mapObservationToObservationStatement(MultiStatementObservationHol
             )
             .ifPresent(narrativeStatementsBlock::append);
 
-        if (observation.hasReferenceRange() && observation.hasValueQuantity()) {
+        if (observation.hasReferenceRange()) {
             Observation.ObservationReferenceRangeComponent referenceRange = observation.getReferenceRangeFirstRep();
 
-            extractUnit(referenceRange)
-                .filter(referenceRangeUnit -> isRangeUnitValid(referenceRangeUnit, observation.getValueQuantity()))
-                .map(RANGE_UNITS_PREFIX::concat)
-                .map(comment ->
-                    mapObservationToNarrativeStatement(holder, comment, CommentType.COMPLEX_REFERENCE_RANGE.getCode())
+            if (observation.hasValueQuantity()) {
+                extractUnit(referenceRange)
+                    .filter(referenceRangeUnit -> isRangeUnitValid(referenceRangeUnit, observation.getValueQuantity()))
+                    .map(RANGE_UNITS_PREFIX::concat)
+                    .map(StringUtils.LF::concat)
+                    .ifPresent(interpretationTextAndComment::append);
+            } else {
+                interpretationTextAndComment
+                    .append(StringUtils.LF)
+                    .append(prepareReferenceRangeToComment(referenceRange));
+            }
+        }
+
+        if (!interpretationTextAndComment.toString().isBlank()) {
```

Comment:
This can also be like this:
```
        Optional.of(interpretationTextAndComment)
            .map(StringBuilder::toString)
            .filter(StringUtils::isNotBlank)
            .map(String::trim)
            .map(textAndComment ->
                mapObservationToNarrativeStatement(holder, textAndComment, prepareCommentType(observation).getCode()))
            .ifPresent(narrativeStatementsBlock::append);
```